src/OrbitGl/TimerTrack.cpp

```diff
@@ -328,9 +328,7 @@ void TimerTrack::OnTimer(const TimerInfo& timer_info) {
 float TimerTrack::GetHeight() const {
   uint32_t collapsed_depth = std::min<uint32_t>(1, GetDepth());
   uint32_t depth = collapse_toggle_->IsCollapsed() ? collapsed_depth : GetDepth();
-  return GetHeaderHeight() + layout_->GetTrackContentTopMargin() +
-         layout_->GetTextBoxHeight() * depth +
-         (depth > 0 ? layout_->GetSpaceBetweenTracksAndThread() : 0) +
```

Comment:
I'm also not sure why we even had this, and if this has any hidden implications. But I guess this is a case of "if it looks correct, it is correct". Did you make sure it looks correct for tracks with and without dynamically instrumented functions?

Reply:
Yes, I checked the capture for the release testing :)